src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java

```diff
@@ -167,6 +167,16 @@ public String getColumnValue(final VariantContext variant) {
         }
     },
 
+    QUALapprox { // Required
+        public String getColumnValue(final VariantContext variant) {
+            String out = getAttribute(variant, "QUALapprox", null);
+            if (out == null) {
+                throw new UserException("Cannot be missing required value for QUALapprox at site: " + variant.toString());
```

Comment:
neat. out of curiosity - is there documentation somewhere of the input requirements? is QUALapprox always present in gvcfs? also thinking more broadly about the method of reblocking required, etc.

Reply:
It's more that we know the downstream tools require this annotation, but it's produced by GATK Reblocking (which is the real prerequisite).